tests/acceptance/expected-failures-webUI-on-OWNCLOUD-storage.txt

```diff
@@ -189,10 +189,6 @@ webUIPreview/imageMediaViewer.feature:33
 # https://github.com/owncloud/web/issues/4296 media viewer does not display jpeg file
 webUIPreview/imageMediaViewer.feature:15
 #
-#
-webUIPreview/imageMediaViewer.feature:100
-webUIPreview/imageMediaViewer.feature:114
```

Comment:
Note: I had accidentally duplicated these in the expected-failures list. They also appear at the end, around lines 423, 424.